state/upgrades.go

```diff
@@ -2348,9 +2348,10 @@ func ChangeSubnetAZtoSlice(pool *StatePool) (err error) {
 			})
 		}
 
-		ops = append(ops, st.createDefaultSpaceOp())
-
-		return errors.Trace(st.db().RunTransaction(ops))
+		if len(ops) > 0 {
```

Comment:
drive by after seeing run transaction with no ops fail msg in controller log on ReplacePortsDocSubnetIDCIDR.